frontend/templates/product/sections/CompatibilityNotesSection/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Heading, SimpleGrid } from '@chakra-ui/react';
+import { Wrapper } from '@ifixit/ui';
+import type { Product } from '@pages/api/nextjs/cache/product';
+
+export type CompatibilityNotesSectionProps = {
+   compatibilityNotes?: Product['compatibilityNotes'] | null;
+};
+
+const MAX_DEFAULT_VISIBLE_DEVICES = 10;
+
+export function SplitCompatibilityNotes({
```

Comment:
I think non-component functions should begin with a lowercase letter.

Reply:
frick i had it right and changed it